config/m4/compiler.m4

```diff
@@ -404,13 +426,19 @@ ADD_COMPILER_FLAG_IF_SUPPORTED([--diag_suppress 1901],
                                [],
                                [])
 
+CHECK_COMPILER_FLAG([-pedantic], [-pedantic],
```

Comment:
pls add comment to it like the rest of the checks

Reply:
added comment